src/test/java/com/amazon/ion/impl/IonRawTextWriterTest_1_1.kt

```diff
@@ -0,0 +1,1022 @@
+            stepOut()
+        }
+    }
+
+    @Test
+    fun `write multiple nested prefixed lists`() {
+        assertWriterOutputEquals("[[[[[]]]]]") {
+            repeat(5) { stepInList(false) }
+            repeat(5) { stepOut() }
+        }
+    }
+
+    @Test
+    fun `write multiple nested delimited lists`() {
+        assertWriterOutputEquals("[[[[]]]]") {
+            repeat(4) { stepInList(true) }
+            repeat(4) { stepOut() }
+        }
+    }
+
+    @Test
+    fun `write multiple nested delimited and prefixed lists`() {
+        assertWriterOutputEquals("[[[[[[[[]]]]]]]]") {
+            repeat(4) {
+                stepInList(true)
+                stepInList(false)
+            }
+            repeat(8) { stepOut() }
+        }
+    }
+
+    @Test
+    fun `write a sexp`() {
+        assertWriterOutputEquals("(true false)") {
+            stepInSExp(delimited = true)
+            writeBool(true)
+            writeBool(false)
+            stepOut()
+        }
+        assertWriterOutputEquals("(true false)") {
+            stepInSExp(delimited = false)
+            writeBool(true)
+            writeBool(false)
+            stepOut()
+        }
+    }
+
+    @Test
+    fun `write multiple nested sexps`() {
+        assertWriterOutputEquals("(((((((())))))))") {
+            repeat(4) {
+                stepInSExp(delimited = true)
+                stepInSExp(delimited = false)
+            }
+            repeat(8) { stepOut() }
+        }
+    }
+
+    @Test
+    fun `write a struct`() {
+        assertWriterOutputEquals(
+            """{$11:true,$12:false}"""
+        ) {
+            stepInStruct(delimited = false)
+            writeFieldName(11)
+            writeBool(true)
+            writeFieldName(12)
+            writeBool(false)
+            stepOut()
+        }
+        assertWriterOutputEquals(
+            """{$11:true,$12:false}"""
+        ) {
+            stepInStruct(delimited = true)
+            writeFieldName(11)
+            writeBool(true)
+            writeFieldName(12)
+            writeBool(false)
+            stepOut()
+        }
+    }
+
+    @Test
+    fun `write multiple nested structs`() {
+        assertWriterOutputEquals(
+            "{a:{b:{a:{b:{a:{b:{a:{b:{}}}}}}}}}"
+        ) {
+            stepInStruct(delimited = false)
+            repeat(4) {
+                writeFieldName("a")
+                stepInStruct(delimited = true)
+                writeFieldName("b")
+                stepInStruct(delimited = false)
+            }
+            repeat(9) {
+                stepOut()
+            }
+        }
+    }
+
+    @Test
+    fun `write empty struct`() {
+        assertWriterOutputEquals("{}") {
+            stepInStruct(delimited = false)
+            stepOut()
+        }
+        assertWriterOutputEquals("{}") {
+            stepInStruct(delimited = true)
+            stepOut()
+        }
+    }
+
+    @Test
+    fun `write prefixed struct with a single flex sym field`() {
```

Comment:
I'd suggest updating the names of these to avoid the binary-specific terminology, unless you think we should just parameterize the existing raw binary writer tests to assert against both the expected binary and expected text, depending on the writer type...